od/audio/WavFileReader.cpp

```diff
@@ -217,6 +218,21 @@ namespace od
         mSampleCount = header.cksize / mChannelCount / mBytesPerChannel;
         data_found = mSampleCount > 0;
       }
+      else if ((header.ckID[0] == 'c' && header.ckID[1] == 'u' && header.ckID[2] == 'e' && header.ckID[3] == ' ') || (header.ckID[0] == 'C' && header.ckID[1] == 'U' && header.ckID[2] == 'E' && header.ckID[3] == ' '))

```

Comment:
https://sites.google.com/site/musicgapi/technical-documents/wav-file-format#cue

I feel like this could be made shorter by using the hex codes, but I followed the existing style.